platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java

```diff
@@ -181,6 +178,26 @@ public void testExponentialBackoff() {
         }
 
         assertThat(isExponentialBackoff(backoffs), is(true));
+        assertThat("List should not contain a value greater than the maximum cutoff value",
+                backoffs,
+                not(hasItemInArray(greaterThan(getMaxTimoutSeconds()))));
```

Comment:
I had a bug where an interval larger than the max got into the array. This was a nice, non-intrusive place to stick a test for it.